Gems/Maestro/Code/Source/Cinematics/AnimNode.h

```diff
@@ -189,20 +190,18 @@ class CAnimNode
     void AnimateSound(std::vector<SSoundInfo>& nodeSoundInfo, SAnimContext& ec, IAnimTrack* pTrack, size_t numAudioTracks);
     //////////////////////////////////////////////////////////////////////////
 
-    int m_refCount;
+    AnimTracks m_tracks;
     AnimNodeType m_nodeType;
-    int m_id;
     AZStd::string m_name;
     IAnimSequence* m_pSequence;
     IAnimNodeOwner* m_pOwner;
     IAnimNode* m_pParentNode;
+    int m_refCount;
+    int m_id;
```

Comment:
It'd be nice if all the member pointers are set to `nullptr`, the `int` members to 0 and the `bool` members to false.